### PR TITLE
Update classic/style.css with TabWidget, TrackView, PianoView and Fader colors

### LINKED ISSUE #3652: Text missing on the envelope tabs with Classic theme.
Since 53772c052ba8d0de1bb82b22249895e4737cfdf6 #3569, `Allow WidgetTab to use artwork tabs (@StCyr)`, some text is missing with the Classic Theme. I've only noticed this on the instrument plugins envelope tab as shown below. 'GENERAL SETTINGS' and the row above the envelopes beginning with 'TARGET' is missing. @Umcaruje 

classic

### PR DESCRIPTION
**Pull request by PhysSong**
It fixes #3652 by adding TabWidget color settings to classic/style.css.
@Umcaruje @RebeccaLaVie If you don't like this color, please change it.

### CODE REVIEW

**Comment by PhysSong** [comment]
@Umcaruje Now you can push commits on my fork. Please change colors appropriately.

**Comment by tresf** [comment]
> @Umcaruje @RebeccaLaVie If you don't like this color, please change it.

@PhysSong Can you please provide a preview of this PR?  That would make the decision a bit quicker.

**Comment by PhysSong** [comment]
@tresf I can't provide preview right now. Currently colors are the same as default theme.

**Comment by Umcaruje** [comment]
I'll look into this today

**Comment by Umcaruje** [comment]
I updated the colors:
image

I also ran a diff between the two themes, and updated some other changes in the classic theme that we missed, so the files now should have the identical rules, just different colors.

**Comment by PhysSong** [comment]
@Umcaruje Great job! Thanks a lot.

**Comment by Umcaruje** [comment]
I'll leave this open for 12 hours if anyone wants to chime in, I'll merge afterwards.

File: data/themes/classic/style.css [hunk 1]
```diff
@@ -24,8 +24,8 @@ AutomationEditor {
 
 	qproperty-graphColor: rgba(153, 175, 255, 200);
 	qproperty-scaleColor: qlineargradient(spread:reflect,
-                     x1:0, y1:0.5, x2:1, y2:0.5,
-                     stop:0 #333, stop:1 #202020);
+				x1:0, y1:0.5, x2:1, y2:0.5,
```
Comment (tresf) [inline data/themes/classic/style.css]: These lines don't appear to have been changed and the formatting is inconsistent.

**Comment by tresf** [comment]
I see a lot of unnecessary formatting changes.  Can we just change the lines that are needed for the PR?  `data/themes/default/style.css` shouldn't even be in this PR.  The more lines we touch, the harder it is to rebase onto master.

**Comment by Umcaruje** [comment]
The thing is that I tried to make the formatting consistent throughout the file, cause it was all over the place, and using spaces instead of tabs. I am not aware of any theme changes on master, so I don't think rebasing this would be a problem.

**Comment by PhysSong** [comment]
@tresf Yes. `default/style.css` shouldn't in this PR.
We need only fix difference between two css file. Formatting is what @Umcaruje should do after this PR is merged.

**Comment by tresf** [comment]
> Formatting is what @Umcaruje should do after this PR is merged.

Formatting should be targeted at master, really and we have an initiative to do formatting cleanup anyway.  Stable branches are not actively developed, so making the source more readable serves no purpose.

**Comment by tresf** [comment]
Link for formatting feature: https://github.com/LMMS/lmms/issues/3071

**Comment by PhysSong** [comment]
@tresf @Umcaruje If there's no objection, I'll revert last three commits and andd a commit to remove unnecessary formatting change.
I will also provide diff file between re-formatted css and one without formatting change. @Umcaruje You will be able to use the diff for further formatting.

**Comment by Umcaruje** [comment]
>@tresf @Umcaruje If there's no objection, I'll revert last three commits and andd a commit to remove unnecessary formatting change.
I will also provide diff file between re-formatted css and one without formatting change. @Umcaruje You will be able to use the diff for further formatting.

Or I can just re-commit the 5 lines I changed tommorow, you two make good points and I'll fix this :)

**Comment by PhysSong** [comment]
@Umcaruje I can do it soon. Would the commits be squashed on merge?
@tresf Thanks for your feedback!

**Comment by Umcaruje** [comment]
>Would the commits be squashed on merge?

Yeah, thanks for working on this. Don't forget the changes in Fader and the track background and the piano widget :)

**Comment by tresf** [comment]
> Would the commits be squashed on merge?

Yes, that is our [default option](https://user-images.githubusercontent.com/6345473/27774124-fce6a350-5f58-11e7-85be-56c862d51cf8.png), so they'd be squashed unless asked otherwise.

**Comment by PhysSong** [comment]
I reverted formatting changes. I also attached the [diff](https://github.com/LMMS/lmms/files/1118172/diff_css.txt) for further formatting.

**Comment by PhysSong** [comment]
Travis CI is failing, but it isn't a problem obviously.
See [this](https://travis-ci.org/LMMS/lmms/jobs/249455504#L2774).

**Comment by zonkmachine** [comment]
> Travis CI is failing, but it isn't a problem obviously.

I restarted the failing build. :white_check_mark:

**Comment by PhysSong** [comment]
@Umcaruje Now fixed.

**Comment by Umcaruje** [comment]
@PhysSong thank you very much for your work on this, this is ready to merge :+1:

**Pull request merged**